<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>추가 학습</string>
  </property>
  <widget class="QLineEdit" name="SrcPath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOpenSrc">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>폴더 열기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>학습 강도</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>학습할 이미지 데이터 위치</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DestPath">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>142</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>112</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>학습 데이터를 저장할 위치</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOpenDest">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>142</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>폴더 열기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnRun">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>190</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>실행</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>캐릭터 이름을 입력하세요</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnNameInput">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>215</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>입력</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineNameInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>214</y>
     <width>290</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelNameInput">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>268</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>현재 입력된 값 :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
